antlr/Excellent1.g4

```diff
@@ -1,71 +1,60 @@
 grammar Excellent1;
```

Comment:
this is the grammar file from rapidpro/expressions so it should work fine with any legacy expression